_maps/RandomRuins/IceRuins/icemoon_surface_engioutpost.dmm

```diff
@@ -182,8 +182,9 @@
 /turf/open/floor/engine/airless,
 /area/icemoon/surface/outdoors)
 "aN" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/brown,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/brown,
```

Comment:
should there really be two pipes of the same color on the same tyle?

Reply:
derp